identifiers/country-my/constituencies.csv

```diff
@@ -0,0 +1,223 @@
+id,name

```

Comment:
I'm a bit torn on whether to leave the nesting as is or to change it.

On one hand, it sounds like the Election Commission of Malaysia is in charge of all redistricting: https://en.wikipedia.org/wiki/Election_Commission_of_Malaysia#Functions

However, it also appears that, by law, each state/territory has a set number of constituencies so they will never cross those boundaries: https://web.archive.org/web/20090726120429/http://www.spr.gov.my/index_files/proses_praya/parlimen_dun.asp

So I'm leaning toward keeping the nesting of the electoral districts under each state, but would be intersted what others think.

Reply:
Maybe @jpmckinney for opinions on the nesting?

Reply:
I think elsewhere we always have national/federal electoral districts hang off the country. In this case, they are also numbered to be unique at that level. If it were like the US where they are numbered within each state, there might be a reason to keep the state level.

Reply:
(This is assuming that the numbers for states and districts are from an official source.)

Reply:
I did check official sources and noticed that they always use the numbers alongside the name, so I decided to keep the numbers as the unique identifiers. See:

Full list: https://www.spr.gov.my/sites/default/files/Senarai%20BPR%2010.09.2019.pdf

Other documents from the official site that list always the number + name:
- [notice of candidate name & official drawing for election #14](https://www.spr.gov.my/sites/default/files/MAKLUMAT%20PUSAT%20PENAMAAN%20CALON%20%28PPC%29%20%20PUSAT%20PENJUMLAHAN%20RASMI%20UNDI%20%28PPRU%29%20PRU%20KE-14%20-%20UPDATE%2018.4.2018_0.pdf)
- [re-examination report for sabah](https://www.spr.gov.my/sites/default/files/Laporan%20Sabah%20FINAL.pdf)
- [other areas of the site](https://www.spr.gov.my/ms/infografik-pilihan-raya)